docs/reference/operator_parameters.md

```diff
@@ -394,6 +394,20 @@ yet officially supported.
   present and accessible by Postgres pods. At the moment, supported services by
   Spilo are S3 and GCS. The default is empty.
 
+* **wal_gs_bucket**
+  GCS bucket to use for shipping WAL segments with WAL-E. A bucket has to be
+  present and accessible by Postgres pods. Note, only the name of the bucket is
+  required. At the moment, supported services by Spilo are S3 and GCS.
+  The default is empty.
+
+* **gcp_credentials**
+  Used to set the GOOGLE_APPLICATION_CREDENTIALS environment variable for the pods.
```

Comment:
I don't immedately see where you in fact use the secret mount part?

Having the secrets in pod env var like this exposes them to a lot of readers, would be a proper secret ref be the better choice.

I thought we support all required google fields already.

Reply:
I thought so too and I could be wrong on what it would take to get this working in GCP without my changes but from reading the documentation it wasn't clear on how to get this to work for GCP.

- GOOGLE_APPLICATION_CREDENTIALS env variable will get populated with a file path where a credentials.json file (or whatever you call it) to the service account who has the IAM rules to push to the GCS configured bucket. This service account is then used be SPILO to push the WAL-E information the GCS bucket. 

- We then use the file mount features to mount the credientails.json to the same path as the path specified by the GOOGLE_APPLICATION_CREDENTIALS 

We are using this in production for our applications in GCP. Would it be helpful on giving an example on how we did this?

Reply:
Yes, it might be worth to add a sub-chapter [here](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#access-to-cloud-resources-from-clusters-in-non-cloud-environment) in the admin docs. Wondering if #946 could also be used in the future, but for now, let's take the setup which you use in production